apps/s_time.c

```diff
@@ -310,7 +310,6 @@ int s_time_main(int argc, char **argv)
     }
     totalTime += tm_Time_F(STOP); /* Add the time for this iteration */
 
-    i = (int)((long)time(NULL) - finishtime + maxtime);
```

Comment:
you need to remove the declaration of i from the function as well, now that its not used anywhere, or you will get a build break when compiling with --strict-warnings